fast64_internal/f3d/f3d_writer.py

```diff
@@ -2336,6 +2336,19 @@ def saveOrGetTextureDefinition(fMaterial, fModel, image: bpy.types.Image, imageN
     print("Finished converting.")
     fModel.addTexture((image, (texFormat, "NONE")), fImage, fMaterial)
 
+    if bpy.context.scene.fast64.oot.shipOfHarkinianCompatible:
```

Comment:
For texture endian issues, instead of doing this, check  `oot_model_classes.py -> OOTGfxFormatter -> __init__`, you can conditionally change the `texArrayBitSize` argument to `8` which should save the texture array as a `u8` instead of a `u64`.